modules/programs/firefox/default.nix

```diff
@@ -33,6 +33,7 @@ in
       };
       platforms.darwin = {
         configPath = "Library/Application Support/Firefox";
+        defaultsId = "org.mozilla.firefox.plist";
```

Comment:
Doesn't seem to handle different versions of firefox. 

I had to change it to `org.nixos.firefoxdeveloperedition` to work in my setup.

Reply:
i've now resolved it to be something defined in the mkFirefoxModule but also a user option that can be changed to allow for scenarios like yours